src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -7,19 +7,32 @@
 import com.chopshop166.chopshoplib.PersistenceCheck;
 import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
 
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.maps.subsystems.ArmMap;
 import frc.robot.maps.subsystems.ArmMap.Data;
 
 public class Arm extends SmartSubsystemBase {
 
     public Data data = new Data();
-    public ArmMap map;
+    public ArmMap extendMap;
     public final double SPEED = 0.3;
     private final double RETRACT_SPEED = -0.1;
+    final double PIVIOT_HEIGHT = 46.654;
+    private double ARM_ANGLE;
+
+    NetworkTableInstance inst = NetworkTableInstance.create();
```

Comment:
Can this be done on startup? Or does it have to wait for something else to come up? Also you're creating an entire table?

Reply:
I *highly* recommend using `getDefault` here, per [the documentation](https://docs.wpilib.org/en/stable/docs/software/networktables/tables-and-topics.html)